index.js

```diff
@@ -0,0 +1,8 @@
+const server = require('./api/server.js');
+require('dotenv').config();
+
+
+const PORT = process.env.PORT || 3000;
+server.listen(PORT, () => {
+  console.log(`Server listening on port ${PORT}`);
```

Comment:
it's possible to define a PORT in the .env file, but you shouldn't specify the PORT when deployed to heroku since they determine that for you.